compaction/compaction.cc

```diff
@@ -811,7 +812,8 @@ class compaction {
                         max_purgeable_func(),
                         _table_s.get_tombstone_gc_state(),
                         get_compacted_fragments_writer(),
-                        get_gc_compacted_fragments_writer());
+                        get_gc_compacted_fragments_writer(),
+                        &_rows_merged);
```

Comment:
see that we missed make_compacting_reader() in consume_without_gc_writer(). I think we can add a test with TimeWindow usage.